source/about_jena/security-advisories.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Jena Security Advisories
+---
+
+# Security Issue Policy
+
+## Process
+
+Jena follows the standard [ASF Security for Committers](https://www.apache.org/security/committers.html) policy for
+reporting and addressing security issues.
+
+If you think you have identified a Security issue in our project please refer to that policy for how to report it, and
+the process that the Jena Project Management Committee (PMC) will follow in addressing the issue.
+
+## Single Supported Version
+
+As a project with a relatively small developer community Apache Jena only has the resources to maintain a single release
+version.  Therefore any accepted security issue reported will be fixed by developing a fix for our `main` branch.  We
+will then make a release with the fix in a timeframe appropriate to the severity of the issue.  
```

Comment:
```suggestion
version.  Any accepted security issue will be fixed in a future release in a timeframe appropriate to the severity of the issue.  
```
We're suppose to not highlight security code fixes until the CVE is public (e.g. no git commit reference). Being git, we can't have private branches built no need to highlight the development.